dockerfiles/che/entrypoint.sh

```diff
@@ -311,7 +311,11 @@ add_cert_to_truststore() {
   echo yes | keytool -keystore $JAVA_TRUST_STORE -importcert -alias "$2" -file $SELF_SIGNED_CERT -storepass $DEFAULT_JAVA_TRUST_STOREPASS > /dev/null
   # set read-only permissions on keystore file
   chmod 444 "$JAVA_TRUST_STORE"
-  if [[ "$JAVA_OPTS" != *"-Djavax.net.ssl.trustStore"* && "$JAVA_OPTS" != *"-Djavax.net.ssl.trustStorePassword"* ]]; then
+  set_truststore_system_variables
+}
+
+set_truststore_system_variables() {
```

Comment:
I think for some cases we will have duplication in JAVA_OPTS.

Reply:
but it has a check